### PR TITLE
Update threat-analytics.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Required permissions and roles to access threat analytics

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7094

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/security/defender-endpoint/threat-analytics.md [hunk 1]
```diff
@@ -51,6 +51,12 @@ Watch this short video to learn more about how threat analytics can help you tra
...
+The following table outlines the roles and permissions required to access Threat Analytics. Roles defined in the table below refer to custom roles in individual portals and are not connected to global roles in Azure AD, even if similarly named.
+
+| **One of the following roles are required for Microsoft 365 Defender**  | **One of the following roles are required for Defender for Endpoint**  | **One of the following roles are required for Defender for Office 365** | **One of the following roles are required for Defender for Cloud Apps** | 
+|---------|---------|---------|---------|
+| Threat Analytics | Alerts and incidents data: <ul><li>View data- security operations</li></ul>TVM mitigations:<ul><li>View data - Threat and vulnerability management</li></ul> | Alerts and incidents data:<ul> <li>View-only Manage alerts</li> <li>Manage alerts</li> <li>Organization configuration</li><li>Audit logs</li> <li>View-only audit logs</li><li>Security reader</li> <li>Security admin</li><li>View-only recipients</li> </ul> Prevented email attempts: <ul><li>Security reader</li> <li>Security admin</li><li>View-only recipients</li> | Not available for Defender for Cloud Apps or MDI users |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/threat-analytics.md]:
```suggestion
| Threat Analytics | Alerts and incidents data: <ul><li>View data- security operations</li></ul>TVM mitigations:<ul><li>View data - Threat and vulnerability management</li></ul> | Alerts and incidents data:<ul> <li>View-only manage alerts</li> <li>Manage alerts</li> <li>Organization configuration</li><li>Audit logs</li> <li>View-only audit logs</li><li>Security reader</li> <li>Security admin</li><li>View-only recipients</li> </ul> Prevented email attempts: <ul><li>Security reader</li> <li>Security admin</li><li>View-only recipients</li> | Not available for Defender for Cloud Apps or MDI users |

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copt review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this PR. Thanks

**Comment by schmurky** [comment]
Hi @DulceMontemayor can you check, this is regarding TA permissions.

**Comment by yogkumgit** [comment]
@DulceMontemayor Please

**Comment by DulceMontemayor** [comment]
@yogkumgit i need to check this with the PM as the MDE and M365D experience might be a bit different. I'll get back to you as soon as i get his feedback

**Comment by yogkumgit** [comment]
@DulceMontemayor Awaiting your kind response. Thanks

**Review by DulceMontemayor — APPROVED** [review]

**Comment by DulceMontemayor** [comment]
@yogkumgit i've approved it but pls update the branch because it's out of date with the base branch. Thanks!

**Comment by msbemba** [comment]
@DulceMontemayor i have updated the branch. Thank you

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**